tests/console/gdb.pm

```diff
@@ -39,18 +36,15 @@ sub wait_serial_or_die {
 
 
 sub run {
-    #Setup console for text feedback.
-    select_console("root-console");
-    if (is_sle('=12-SP5') && is_aarch64()) {
```

Comment:
Where did this go?

Reply:
This was some leftover from past, not needed at all.

Reply:
Should be part of a separate commit still

Reply:
Yes, but as I went trough the tests, I was cleaning up. I brushed this not needed code with the rest. I agree and will separate it when you insist. :innocent: